jota_utils/dict.py

```diff
@@ -52,3 +52,19 @@ def json_loads(json_string, default=None):
         return json.loads(json_string)
 
     return default
+
+
+def with_except(obj, keys):
```

Comment:
É possível conseguir uma performance maior utilizando dict comprehension, pra isso basta inverter as funções:

with_only: return {key: value for key, value in obj.items() if key in keys}

with_except: return with_only(obj, set(obj.keys()) - set(keys))

Faz sentido?

Reply:
Faz sim, ficou mais simples.